doc/content/source/bcs/SecondaryElectronEnergyBC.md

```diff
@@ -1,20 +1,41 @@
 # SecondaryElectronEnergyBC
 
-!alert construction title=Undocumented Class
-The SecondaryElectronEnergyBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/SecondaryElectronEnergyBC
 
 ## Overview
 
-!! Replace these lines with information regarding the SecondaryElectronEnergyBC object.
+Note: add $n_{\gamma}$
+
+`SecondaryElectronEnergyBC` is an electron mean energy density of secondary electrons induced by ion flux outflow boundary condition.
+`SecondaryElectronEnergyBC` assumes the electrostatic approximation for the electric field.
+
+\begin{equation}
+a =
+\begin{cases}
+1, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} > 0\\
+0, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \leq 0\\
+\end{cases} \\[10pt]
+\Gamma_{\epsilon} \cdot \textbf{n} = \frac{5}{3} \frac{-2}{1+r} (1 - a_{e}) \gamma \Gamma_{j} \cdot \textbf{n} \\[10pt]
+\Gamma_{j} \cdot \textbf{n} = \frac{1-r_{j}}{1+r_{j}} \left[ (2 a_{j}-1) \ \mu_{j} \text{-} \nabla (V) \cdot \textbf{n} \ n_{j} + \frac{1}{2}v_{th,j}n_{j} \right] \\[10pt]
+v_{th,j} = \sqrt{\frac{8k_{B}T}{\pi m_{j}}}
+\end{equation}
+
+Where $\Gamma$ is the flux of the electron or ions, $\textbf{n}$ is the normal of the boundary,
+$\mu$ is the mobility coefficient, $n$ is the specie density, $v_{th}$ is the thermal velocity of the species, $k_{B}$ is the Boltzmann constant, and $T$ is the gas temperature. $a$ is defined such that the outflow is only defined when the drift velocity is direct towards the wall and zero otherwise. $r$ is defined as the fraction of particles reflected by the surface. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SecondaryElectronEnergyBC` is defined as
```

Comment:
```suggestion
Where $\Gamma$ is the flux of the electron or ions, $\textbf{n}$ is the normal vector of the boundary,
$\mu$ is the mobility coefficient, $n$ is the species density, $v_\text{th}}$ is the thermal velocity of the species, $k_{B}$ is the Boltzmann constant, and $T$ is the gas temperature. $a$ is defined such that the outflow is non-zero when the drift velocity is directed towards the wall and zero otherwise. $r$ is defined as the fraction of particles reflected by the surface. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SecondaryElectronEnergyBC` is defined as
```